ircstates/server.py

```diff
@@ -747,6 +761,30 @@ def _handle_whoisuser(self, line: Line) -> Emit:
             user.realname = realname
         return emit
 
+    @line_handler(RPL_WHOISOPERATOR)
+    def _handle_whoisoper(self, line: Line):
+        emit = self._emit()
+        nickname = line.params[1]
+        lowered = self.casefold(nickname)
+        if lowered is None:
+            lowered = nickname
+
+        if lowered == self.nickname_lower:
+            emit.self = True
+            self.is_oper = True
+
+        if lowered in self.users:
+            self.users[lowered].is_oper = True
```

Comment:
I believe there's no way of knowing when another user stops being oper, so I'd be inclined to not track this at all

Reply:
same goes for `user.oper_name`

Reply:
Yep, thinking about it more theres no way to reliably say "is not", though it seems that the absence of the tag could work assuming we expect it. Ill remove it tomorrow

Reply:
removed